src/state/queries/actor-autocomplete.ts

```diff
@@ -23,11 +23,17 @@ export function useActorAutocompleteQuery(
   const moderationOpts = useModerationOpts()
   const {getAgent} = useAgent()
 
-  prefix = prefix.toLowerCase()
+  const prefixTrimmed = React.useMemo(() => {
+    let trimmed = prefix.toLowerCase().trim()
+    if (trimmed.endsWith('.')) {
```

Comment:
Why special-case a single period? Does it not happen with other punctuation or more than one period?

Reply:
I think a trailing period is the only thing I'd expect to still have a valid result. I.e.

- `pfrazee..` would be invalid
- `pfrazee,` would be invalid